fluent/src/resolver.js

```diff
@@ -79,8 +106,13 @@ function DefaultMember(env, members, def) {
 
 
 /**
- * Resolve a reference to a message to the message object.
+ * Resolve a reference to another message.
  *
+ * @param   {Object} env
+ *    Resolver environment object.
+ * @param   {String} name
+ *    ID of a message for to be resolved.
```

Comment:
I don't fully like it because there's no "id" in the function signature, but I'll follow along.